src/originalBudgetTrackingApp/export/outputVendors/csv/csv.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  EnrichedTransaction, ExportTransactionsFunction, OutputVendor, OutputVendorName
+} from '@/originalBudgetTrackingApp/commonTypes';
+import { mergeTransactions, sortByDate } from '@/originalBudgetTrackingApp/transactions/transactions';
+import { TransactionInstallments } from '@brafdlog/israeli-bank-scrapers-core/lib/transactions';
+import { promises as fs } from 'fs';
+
+const stringify = require('csv-stringify/lib/sync');
+const parse = require('csv-parse/lib/sync');
+
+export function parseTransactions(csvText: string) {
+  return parse(csvText, {
+    columns: true,
+    cast: (value, context) => {
+      switch (context.column) {
+        // note: we check here some number-type values.
+        // if there are undefined string-type values (memo?, for example) they'll be serialized as empty string.
+        // it means that the deserialized object will not be 100% identical to the serialzed.
+        // I'm not sure how to represent empty string differently than undefined string.
```

Comment:
If you had tests on these cases, I would understand more what you expect.

(you have tests, but the tests are not including the cases of empty string or undefined, and I think in this case it is easy to understand your solution from test examples, instead of trying to read the code.)

Reply:
Ok, I will remove from the main code and add a short explanation in the test.